HANDSON3.md

```diff
@@ -17,8 +17,9 @@ After entering the required information, click **Crete repository from template*
 Let's clone a forked repository
 
 ```sh
-git clone  https://github.com/YOUR-USERNAME/REPOSITORY-NAME smarttest-workshop
-cd smarttest-workshop
+git clone  https://github.com/YOUR-USERNAME/REPOSITORY-NAME smarttests-workshop
+cd smarttests-workshop
+git switch -c launchable-test
```

Comment:
This action runs only PR and dispatch

https://github.com/launchableinc/hands-on-lab/blob/main/.github/workflows/pre-merge.yml#L4-L5